site/src/pages/WorkspacePage/WorkspacePage.tsx

```diff
@@ -49,27 +49,29 @@ export const WorkspacePage: FC = () => {
   });
   const permissions = permissionsQuery.data as WorkspacePermissions | undefined;
 
-  // Builds
-  const buildsQuery = useInfiniteQuery({
-    ...infiniteWorkspaceBuilds(workspace?.id ?? ""),
-    enabled: workspace !== undefined,
-  });
-
   // Watch workspace changes
   const updateWorkspaceData = useEffectEvent(
     async (newWorkspaceData: Workspace) => {
+      if (!workspace) {
```

Comment:
This workspace is a prop from the WorkspacePage so I don't think renaming it would make it look good at the component level. Eg. `<WorkspacePage currentWorkspace={workspace} />` does not make too much sense to me.